<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt4CTestSuiteBis</class>
 <widget class="QWidget" name="Qt4CTestSuiteBis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>tot_counter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit1"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>tot_positive</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>tot_negative</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEdit3"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox4">
     <property name="title">
      <string>enable_pos_log</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonApply4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>toggle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox7">
     <property name="title">
      <string>enable_library</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButtonApply7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>toggle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox8">
     <property name="title">
      <string>disable_library</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButtonApply8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>toggle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox5">
     <property name="title">
      <string>disable_pos_log</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="pushButtonApply5">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>toggle</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox9">
     <property name="title">
      <string>n</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEdit9"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox6">
     <property name="title">
      <string>counter</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLineEdit" name="lineEdit6"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonNext">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
